atomic_reactor/plugins/exit_koji_tag_build.py

```diff
@@ -74,6 +75,14 @@ def run(self):
             self.log.info('Build failed, skipping koji tagging')
             return
 
+        if is_scratch_build(self.workflow):
+            self.log.info('scratch build, skipping plugin')
+            return
+
+        if not self.target:
+            self.log.info('no koji target, skipping build')
```

Comment:
Do you think it is worth to say "no koji target provided" here and in the other plugins where a issing param would cause the plugin to skip execution?

Reply:
sure